annotations/fenix/metrics/top_sites.open_baidu_search_attribution/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - Tabs
+---
+
+This is a stub commentary for the `top_sites.open_baidu_search_attribution` metric: please feel free to edit (read the
+[contributing guidelines](https://github.com/mozilla/glean-annotations/blob/main/CONTRIBUTING.md)
+if you haven't done this before)
```

Comment:
When #67 is merged (should be very soon) there won't be any need for stub commentary:

```suggestion
```